index.html

```diff
@@ -248,10 +246,59 @@ <h2 class="about_me_title">About me.</h2>
                 </li>
                 <li class="language">Database Mangement</li>
                 <li class="language">Version Control</li>
-                
+
                 <li class="language">CLI</li>
                 <li class="language">API Design</li>
                 <li class="language">Web Development</li>
             </ul>
   </div>
  </section>
+
+
+
+    <section id="form_container_link" class="form_section">
+      <div class="form_container">
+        <h3 class="form_text">I'm always interested in hearing about new projects, so if you'd like to chat please get in</h2>
+
+        <form id="form_contact" class="contact_me_form" action="https://formspree.io/f/mnqygakk" method="POST">
+
+            <input class="form_input_fnames" form="contact_form" id="name" name="name" type="text" placeholder="Full names" maxlength="30" required>
+            <hr>
+
+            <input class="form_input_lnames" form="contact_form" name= "last_name" type="text" placeholder="Last name" maxlength="30" required>
+            <hr class="lname_line">
+
+            <input class="form_input_email" id="email" name="email" form="contact_form" type="email" placeholder="Email address" required>
+            <hr>
```

Comment:
The validations on the input tags of the form are not working. 🤔

Users should not be able to submit the form if any of the input fields are empty. However, I was able to submit the form by filling on the `Message` fields while all other fields were empty.  please kindly fix this issue 🤷‍♂️

Please let's adjust the alignment of the input fields `placeholder` text to match the template on Figma

Your design
![image](https://user-images.githubusercontent.com/78251927/228388573-57d2d551-4121-4707-9efb-63ad6c72abff.png)

Figma
![image](https://user-images.githubusercontent.com/78251927/228388625-66fe584e-33b5-40b5-9c25-68619d37211f.png)

According to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md), we should stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).